lib/loaders/sass.js

```diff
@@ -35,10 +37,10 @@ module.exports = {
 
         sassLoaders.push({
             loader: 'sass-loader',
-            options: {
+            options: Object.assign(sassOptions, {
```

Comment:
we should not alter the sassOptions object passed as argument. It could have side effects. Use this instead: `Object.assign({}, sassOptions, { sourceMap: ... })` instead